docs/src/add-services/elasticsearch.md

```diff
@@ -22,28 +23,48 @@ See the [Elasticsearch documentation](https://www.elastic.co/guide/en/elasticsea
 
 ## Supported versions
 
-Due to a licensing change, Elasticsearch versions after 7.10 aren't supported.
-For newer versions, use [OpenSearch](./opensearch.md) instead.
-To switch to OpenSearch, follow the same procedure as for [upgrading](#upgrading).
+Elasticsearch is now a premium service.
+This means that from version 7.17 onwards, you need to add Elasticsearch to your project at an additional cost.
```

Comment:
It's >= 7.11 or > 7.10. As it, we could think that 7.11, 7.12 up to 7.16 are free of charge.